multipath/custom.conf

```diff
@@ -0,0 +1,6 @@
+# Custom configuration for multipathd
+
+# Changes made to this file will not be overwritten by future system updates.
+# They will also be retained through system upgrades to newer releases.
+
+# Refer to "man multipath.conf"
```

Comment:
Ideally, this file should contain a commented out skeleton, for easier customization. I don't master multipath configuration, so I haven't done it, but I'm sure @MarkSymsCtx, @Fohdeesha, or someone else will be able to come up with something if my idea is accepted.